test/relay.transaction.test.ts

```diff
@@ -0,0 +1,58 @@
+import { RelayingServices, SmartWallet } from '../src';
+import { MockRelayingServices, Web3Mock } from './mock';
+import { TransactionConfig, TransactionReceipt } from 'web3-core';
+import Expect = jest.Expect;
+import {
+    MOCK_CONTRACT_ADDRESS,
+    MOCK_ADDRESS,
+    MOCK_SMART_WALLET_ADDRESS,
+    MOCK_TRANSACTION_RECEIPT
+} from './constants';
+
+declare const expect: Expect;
+const transaction: TransactionConfig = {
+    from: MOCK_ADDRESS,
+    to: MOCK_CONTRACT_ADDRESS,
+    value: 1
+};
+const smartWallet: SmartWallet = {
+    address: MOCK_SMART_WALLET_ADDRESS,
+    index: 0,
+    deployed: true
+};
+describe('Tests for relay transaction', () => {
+    let sdk: RelayingServices;
+
+    beforeEach(async () => {
+        sdk = new MockRelayingServices();
+        sdk.initialize({});
+    });
+
+    it('Should relay transaction successfully', async () => {
+        const transactionReceipt: TransactionReceipt =
+            await sdk.relayTransaction(transaction, smartWallet, 0);
+        expect(transactionReceipt).toBe(MOCK_TRANSACTION_RECEIPT);
+    });
+});
+
+describe('Tests for smart wallet without being deployed', () => {
+    let sdk: RelayingServices;
+
+    beforeEach(async () => {
+        sdk = new MockRelayingServices(
+            new Web3Mock({
+                getCodeEmpty: true
+            }) as any
+        );
+    });
+
+    it('Should fail to relay transaction', async () => {
+        try {
+            await sdk.relayTransaction(transaction, smartWallet, 0);
```

Comment:
Same as here: https://github.com/infuy/relaying-services-sdk/pull/2/files#r701722807

Reply:
I don't think this can considered as solved as it is